### PR TITLE
Install pkg-config file on non-Unix platforms

### PR DESCRIPTION
**Pull request by Hello71**
This is standard practice; pkg-config is fully usable on Windows too. MXE generates its own pkg-config file, which is incompatible with zstd's.

I believe this change is not copyrightable or patentable in any jurisdiction, but to the extent that it is, all such rights are released under the CC0 license.

### CODE REVIEW

**Comment by Hello71** [comment]
Tor for example uses only pkg-config to locate zstd.

**Comment by Cyan4973** [comment]
Thanks for proposed patch @Hello71 .

Minor detail : could you please direct your patch at the `dev` branch instead ?
Our process disallow direct commit to `master` branch.

**Comment by Hello71** [comment]
I mean, it's the same patch, but OK, whatever.

**Comment by Cyan4973** [comment]
Sure ! It's a trivial patch !
Note that the [CLA]( https://code.facebook.com/cla) need to be signed before a merge.

Alternatively, I can manually reproduce your patch if you prefer.

**Comment by Hello71** [comment]
I mainly wish to avoid an agreement that applies to any future contributions. I don't think "manually reproducing" satisfies copyright or patent law, but I don't think any of this is copyrightable or patentable anyways.

If it helps, I agree to the CLA with respect to this present contribution (i.e. relating to installing pkg-config files on non-Unix platforms) but not with respect to future contributions. All license grants for this contribution are still perpetual and irrevocable though, as already released by my CC0 declaration.

File: build/cmake/lib/CMakeLists.txt [hunk 1]
```diff
@@ -157,20 +157,18 @@ IF (ZSTD_BUILD_STATIC)
             OUTPUT_NAME ${STATIC_LIBRARY_BASE_NAME})
 ENDIF (ZSTD_BUILD_STATIC)
 
-IF (UNIX)
```
Comment (tonytheodore) [inline build/cmake/lib/CMakeLists.txt]: A standard way is `IF (UNIX or MINGW)` to avoid installing for MSVC.

**Pull request closed without merging**